docs/core/porting/third-party-deps.md

```diff
@@ -42,21 +44,20 @@ The easiest way to inspect NuGet package folders is to use the [NuGet Package Ex
 4. Select the package name from the search results and click **open**.
 5. Expand the *lib* folder on the right-hand side and look at folder names.
 
-Look for a folder with names using one the following patterns: `netstandardX.Y` or `netcoreappX.Y`.
+Look for a folder with names using one the following patterns: `netstandardX.Y`, `netX.Y`, or `netcoreappX.Y`.
 
-These values are the [Target Framework Monikers (TFMs)](../../standard/frameworks.md) that map to versions of [.NET Standard](../../standard/net-standard.md), .NET Core, and traditional Portable Class Library (PCL) profiles that are compatible with .NET Core.
+These values are the [Target Framework Monikers (TFMs)](../../standard/frameworks.md) that map to versions of [.NET Standard](../../standard/net-standard.md), .NET, and .NET Core, which are all compatible with .NET.
 
 > [!IMPORTANT]
-> When looking at the TFMs that a package supports, note that `netcoreapp*`, while compatible, is for .NET Core projects only and not for .NET Standard projects.
-> A library that only targets `netcoreapp*` and not `netstandard*` can only be consumed by other .NET Core apps.
+> When looking at the TFMs that a package supports, note that a TFM other than `netstandard*` targets a specific implementation of .NET, such as .NET 5, .NET Core, or .NET Framework.
```

Comment:
Do you want to add a sentence here in that ".net*" effectively replaces "netstandard*" for libraries in the .net5 + world?

Reply:
Great point!

Reply:
```suggestion
> When looking at the TFMs that a package supports, note that a TFM other than `netstandard*` targets a specific implementation of .NET, such as .NET 5, .NET Core, or .NET Framework. Starting with .NET 5, the `net*` TFM (without an operating system) effectively replaces `netstandard*` as a [portable target](../../standard/net-standard.md#net-5-and-net-standard). For example, `net5.0` targets the .NET 5 API surface and is portable and cross-platform friendly, but `net5.0-windows` targets the .NET 5 API surface as it's implemented on the Windows operating system.
```